frontend/lib/norent/letter-builder/welcome.tsx

```diff
@@ -11,50 +11,70 @@ import { hasNorentLetterBeenSentForThisRentPeriod } from "./step-decorators";
 import { li18n } from "../../i18n-lingui";
 import { t, Trans } from "@lingui/macro";
 
-const WelcomePage: React.FC<ProgressStepProps> = (props) => (
-  <Page
-    title={li18n._(t`Build your letter`)}
-    className="content"
-    withHeading="big"
-  >
-    <p>
-      <Trans id="norent.introductionToLetterBuilderSteps">
-        In order to benefit from the eviction protections that local elected
-        officials have put in place, you should notify your landlord of your
-        non-payment for reasons related to COVID-19.{" "}
-        <span className="has-text-weight-semibold">
-          In the event that your landlord tries to evict you, the courts will
-          see this as a proactive step that helps establish your defense.
-        </span>
-      </Trans>
-    </p>
-    <Trans id="norent.outlineOfLetterBuilderSteps">
-      <p>
-        In the next few steps, we’ll build your letter using the following
-        information. Have this information on hand if possible:
-      </p>
-      <ul>
-        <li>
-          <p>your phone number, email address, and residence</p>
-        </li>
-        <li>
+const WelcomePage: React.FC<ProgressStepProps> = (props) => {
+  const { session } = useContext(AppContext);
+  return (
+    <Page
+      title={
+        session.phoneNumber
+          ? li18n._(t`Welcome back!`)
+          : li18n._(t`Build your letter`)
+      }
+      className="content"
+      withHeading="big"
+    >
+      {session.phoneNumber ? (
+        <p>
+          <Trans>
+            Look's like you've been here before. Click "Next" to be taken to
```

Comment:
I think this is a typo?  Should it be "Looks" instead of "Look's"?

Reply:
Good catch!